connectors/utils/table_printer.py

```diff
@@ -0,0 +1,17 @@
+from typing import List, Tuple
+from tabulate import tabulate
+
+
+def print_truncated_table(rows: Tuple[List[str]], columns: List[str], max_rows=6, max_columns=6):
+    if len(rows) > max_rows:
+        final_rows = rows[:max_rows // 2] + tuple([['...' for c in columns]]) + rows[max_rows // -2:]
```

Comment:
If I'm reading this expression right, it's probably better replaced by
```suggestion
        final_rows = [*rows[:max_rows // 2], ['...' for c in columns], *rows[max_rows // -2:]]
```

Reply:
I'm getting `Unpack operation not allowed in comprehension` for that

Reply:
What's saying that?

Reply:
I had VSCode plugin named Pylance, which insisted that.

I disabled the plugin and the code ran fine :+1: